### ISSUE TITLE
Advice when verification of fingerprints has to be done

### ISSUE THREAD

**Issue by claucece** [otrv4/otrv4-client-imp-recommendations#14]
Define that every time that a long-term public key changes, a verification has to happen.

**Comment by olabini** [comment]
I'm not sure what verification means in this case.

**Comment by claucece** [comment]
It means 'manual fingerprint verification' or the SMP. I means verifying the long-term public key and the long-term forging key.

**Comment by olabini** [comment]
Ah ok - cool. So basically this is saying to clients something like "if the known fingerprint for a peer changes at any time, it is highly advisable to ask the end user to re-verify that peer", or something like that?

**Comment by claucece** [comment]
> Ah ok - cool. So basically this is saying to clients something like "if the known fingerprint for a peer changes at any time, it is highly advisable to ask the end user to re-verify that peer", or something like that?

Precisely. And why it is needed: for example, saying that this is needed for online deniability or to protect against identity misbinding. 

This is because some clients don't care about how to verify the long-term public keys, or do it in strange ways: TOFU, scanning of codes. We should say that 'manual fingerprint verification' or the SMP' are the best options, as well.
